salt/utils/gitfs.py

```diff
@@ -528,6 +527,40 @@ def _get_per_saltenv(tgt_env):
                     else:
                         return None
 
+                if self.opts.get("__git_pillar", False) is True:
```

Comment:
Using "truthiness" is preferred:
```suggestion
                if self.opts.get("__git_pillar", False):
```